.travis.yml

```diff
@@ -1,34 +1,45 @@
+sudo: false
 language: cpp
 compiler:
   - clang
   - gcc
 env:
-  - TARGET=debug
-  - TARGET=debug.nounity
-  # We can specify any combination of builds here, for example, to
-  # include release builds, too, uncomment the following lines.
-  #- TARGET=release
-  #- TARGET=release.nounity
+  global:
```

Comment:
I love this!

This is getting a little long, perhaps add whitespace between sections like this example:  https://github.com/rec/echomesh/blob/master/code/python/echomesh/settings/settings.yml